PowerEditor/src/MISC/Common/Common.h

```diff
@@ -188,7 +188,12 @@ generic_string PathAppend(generic_string &strDest, const generic_string & str2ap
 COLORREF getCtrlBgColor(HWND hWnd);
 generic_string stringToUpper(generic_string strToConvert);
 generic_string stringReplace(generic_string subject, const generic_string& search, const generic_string& replace);
-std::vector<generic_string> stringSplit(const generic_string& input, generic_string delimiter);
-generic_string stringJoin(const std::vector<generic_string> &strings, generic_string separator);
+std::vector<generic_string> stringSplit(const generic_string& input, const generic_string &delimiter);
+generic_string stringJoin(const std::vector<generic_string> &strings, const generic_string &separator);

```

Comment:
`const std::vector<generic_string>& input` or `const std::vector<generic_string> &input`?

Consistency is good.